engine/build.gradle

```diff
@@ -156,6 +156,7 @@ dependencies {
 
     // Wildcard dependency to catch any libs provided with the project (remote repo preferred instead)
     implementation fileTree(dir: 'libs', include: '*.jar')
+    apiElements fileTree(dir: 'libs', include: '*.jar')
```

Comment:
What's this apiElements line about?

Should we change the above `implementation` line to an `api` instead?

Reply:
Next PR(also rename error message about BasicRendering)
Returning to small PRs.